examples/dell5k_edge_system_iotcc.py

```diff
@@ -69,7 +69,7 @@ def read_disk_usage_stats():
 
 
 def read_network_bytes_received():
-    return round(net_stat.rx_tx_bytes('eth0')[0], 2)
+    return round(net_stat.rx_tx_bytes('ens33')[0], 2)
```

Comment:
@Venkat2811 As we are using linux_metrics library for the examples, our code can anyway be run for Linux OS alone. Hence, I guess it should be fine to use Linux specific commands. When we introduce example for other OS (Windows), we can rework the example without linux_metrics library and linux specific command.